main.go

```diff
@@ -42,6 +50,42 @@ func withValidProjectURI(r RunE) RunE {
 	}
 }
 
+func checkForUpdates(latestVersion string, currentVersion string) {
+
+	// Don't check for updates if we're running a dev version
+	if latestVersion == "dev" || currentVersion == "dev" {
+		return
+	}
+
+	if latestVersion != currentVersion {
+		ui.Attentionf("Your unweave CLI is out of date. Yours: %s, Latest: %s.", currentVersion, latestVersion)
+		ui.Attentionf("Run `brew upgrade unweave` to update it.")
+	}
+}
+
+func getLatestReleaseVersion(owner, repo string) (string, error) {
+	url := fmt.Sprintf("https://api.github.com/repos/%s/%s/releases/latest", owner, repo)
+
+	resp, err := http.Get(url)
+	if err != nil {
+		return "", err
+	}
+	defer resp.Body.Close()
+
+	body, err := ioutil.ReadAll(resp.Body)
```

Comment:
`ioutil.ReadAll` is deprecated. You should use `io.ReadAll` instead

Reply:
Addressed!